2018-09-13 19:10:13.822 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:10:13.824 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:10:13.867 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:10:13.867 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:10:13.868 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:10:14.998 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.130s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:10:15.001 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:10:16.140 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.139s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:10:16.144 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:10:17.358 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.215s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:10:17.364 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:10:18.582 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.219s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:11:13.827 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:11:13.830 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:11:13.909 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:11:13.910 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:11:13.912 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:11:15.466 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.554s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:11:15.470 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:11:16.911 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.441s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:11:16.915 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:11:18.100 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.185s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:11:18.103 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:11:19.379 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.276s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:12:13.823 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:12:13.826 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:12:13.885 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:12:13.886 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:12:13.888 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:12:15.310 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.423s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:12:15.314 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:12:16.697 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.383s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:12:16.701 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:12:18.004 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.304s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:12:18.011 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:12:19.517 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.507s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:13:13.823 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:13:13.825 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:13:13.872 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:13:13.874 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:13:13.875 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:13:14.762 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 0.887s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:13:14.764 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:13:15.950 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.186s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:13:15.953 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:13:17.006 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.053s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:13:17.010 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:13:18.571 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.561s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:14:13.824 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:14:13.827 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:14:13.888 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:14:13.889 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:14:13.890 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:14:15.084 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.194s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:14:15.087 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:14:16.168 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.081s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:14:16.173 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:14:17.495 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.322s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:14:17.500 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:14:18.797 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.298s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:15:13.821 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:15:13.826 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:15:13.883 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:15:13.885 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:15:13.887 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:15:15.239 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.353s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:15:15.242 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:15:16.441 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.199s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:15:16.445 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:15:17.619 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.175s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:15:17.623 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:15:18.606 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.983s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:13.821 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:16:13.824 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:16:13.880 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:16:13.881 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:16:13.883 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:15.447 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.564s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:15.451 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:16.012 31485 DEBUG cinder.volume.flows.manager.create_volume [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Volume reschedule parameters: True retry: {u'num_attempts': 1, u'backends': [u'localhost.localdomain@lvm#lvm'], u'hosts': [u'localhost.localdomain@lvm#lvm']} get_flow /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:1126
2018-09-13 19:16:16.136 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Flow 'volume_create_manager' (866a8ce4-2c16-4ec7-a5d1-e4c0168a6524) transitioned into state 'RUNNING' from state 'PENDING' _flow_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:145
2018-09-13 19:16:16.145 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeRefTask;volume:create' (9a6b0845-26c6-4f36-8b5b-89cfe20eeb5b) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-13 19:16:16.222 31485 DEBUG oslo_db.sqlalchemy.engines [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-09-13 19:16:16.318 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeRefTask;volume:create' (9a6b0845-26c6-4f36-8b5b-89cfe20eeb5b) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'Volume(_name_id=None,admin_metadata={},attach_status='detached',availability_zone='nova',bootable=False,cluster=<?>,cluster_name=None,consistencygroup=<?>,consistencygroup_id=None,created_at=2018-09-13T23:16:13Z,deleted=False,deleted_at=None,display_description=None,display_name='tempest-VOLUME_SAMPLE-1536880201',ec2_id=None,encryption_key_id=None,glance_metadata=<?>,group=<?>,group_id=None,host='localhost.localdomain@lvm#lvm',id=5449e583-f52c-42c3-8617-fd51d9d9a775,launched_at=None,metadata={},migration_status=None,multiattach=False,previous_status=None,project_id='33775ae47c434316879560b7bcd0d652',provider_auth=None,provider_geometry=None,provider_id=None,provider_location=None,replication_driver_data=None,replication_extended_status=None,replication_status=None,scheduled_at=2018-09-13T23:16:15Z,size=1,snapshot_id=None,snapshots=<?>,source_volid=None,status='creating',terminated_at=None,updated_at=2018-09-13T23:16:15Z,user_id='c1b56a1d5672426f9e3b1fdf25f5593a',volume_attachment=<?>,volume_type=VolumeType(d9c60c05-9188-4fa8-885a-e4f8424aa313),volume_type_id=d9c60c05-9188-4fa8-885a-e4f8424aa313)' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-13 19:16:16.329 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.OnFailureRescheduleTask;volume:create' (8aa4ec33-4486-4282-bee2-80e8274e660c) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-13 19:16:16.336 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.OnFailureRescheduleTask;volume:create' (8aa4ec33-4486-4282-bee2-80e8274e660c) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-13 19:16:16.344 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeSpecTask;volume:create' (78b61fb9-e96e-45e7-9e44-f9e36d234648) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-13 19:16:16.371 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:16.908 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.458s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:16.912 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:16.980 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928 used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:16.984 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:16 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1536880201", "tags": [], "container_format": "bare", "created_at": "2018-09-13T23:11:02Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-13T23:11:02Z", "visibility": "public", "self": "/v2/images/251939be-2d2c-406e-b776-566b7322c928", "min_disk": 0, "protected": false, "id": "251939be-2d2c-406e-b776-566b7322c928", "file": "/v2/images/251939be-2d2c-406e-b776-566b7322c928/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "bd8a53962fd64ffeab7f2932b0e688c5", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:16.986 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:17.015 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:17.017 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:17 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:17.025 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:17.079 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928 used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:17.081 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:17 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1536880201", "tags": [], "container_format": "bare", "created_at": "2018-09-13T23:11:02Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-13T23:11:02Z", "visibility": "public", "self": "/v2/images/251939be-2d2c-406e-b776-566b7322c928", "min_disk": 0, "protected": false, "id": "251939be-2d2c-406e-b776-566b7322c928", "file": "/v2/images/251939be-2d2c-406e-b776-566b7322c928/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "bd8a53962fd64ffeab7f2932b0e688c5", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:17.082 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:17.107 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:17.111 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:17 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:17.118 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:17.797 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:17.801 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:17 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:17.811 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeSpecTask;volume:create' (78b61fb9-e96e-45e7-9e44-f9e36d234648) transitioned into state 'SUCCESS' from state 'RUNNING' with result '{'status': u'creating', 'image_location': (None, None), 'image_id': u'251939be-2d2c-406e-b776-566b7322c928', 'volume_id': u'5449e583-f52c-42c3-8617-fd51d9d9a775', 'volume_size': 1, 'image_service': <cinder.image.glance.GlanceImageService object at 0x7f7ac58eb510>, 'image_meta': {u'status': u'active', u'name': u'tempest-cirros-0.4.0-x86_64-1536880201', u'tags': [], u'container_format': u'bare', u'created_at': datetime.datetime(2018, 9, 13, 23, 11, 2, tzinfo=<iso8601.Utc>), u'disk_format': u'qcow2', u'updated_at': datetime.datetime(2018, 9, 13, 23, 11, 2, tzinfo=<iso8601.Utc>), u'visibility': u'public', 'properties': {}, u'owner': u'bd8a53962fd64ffeab7f2932b0e688c5', u'protected': False, u'id': u'251939be-2d2c-406e-b776-566b7322c928', u'file': u'/v2/images/251939be-2d2c-406e-b776-566b7322c928/file', u'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', u'min_disk': 0, u'virtual_size': None, u'min_ram': 0, u'size': 12716032}, 'volume_name': u'volume-5449e583-f52c-42c3-8617-fd51d9d9a775', 'type': 'image'}' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-13 19:16:17.821 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.NotifyVolumeActionTask;volume:create, create.start' (e43ae92f-65d5-4b09-a7f0-003ee6914d04) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-13 19:16:18.340 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.NotifyVolumeActionTask;volume:create, create.start' (e43ae92f-65d5-4b09-a7f0-003ee6914d04) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-13 19:16:18.345 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeFromSpecTask;volume:create' (6916758e-a9f1-4b20-a04d-c2a301ef4cc5) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-13 19:16:18.347 31485 INFO cinder.volume.flows.manager.create_volume [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Volume 5449e583-f52c-42c3-8617-fd51d9d9a775: being created as image with specification: {'status': u'creating', 'image_location': (None, None), 'volume_size': 1, 'volume_name': u'volume-5449e583-f52c-42c3-8617-fd51d9d9a775', 'image_id': u'251939be-2d2c-406e-b776-566b7322c928', 'image_service': <cinder.image.glance.GlanceImageService object at 0x7f7ac58eb510>, 'image_meta': {u'status': u'active', u'name': u'tempest-cirros-0.4.0-x86_64-1536880201', u'tags': [], u'container_format': u'bare', u'created_at': datetime.datetime(2018, 9, 13, 23, 11, 2, tzinfo=<iso8601.Utc>), u'disk_format': u'qcow2', u'updated_at': datetime.datetime(2018, 9, 13, 23, 11, 2, tzinfo=<iso8601.Utc>), u'visibility': u'public', 'properties': {}, u'owner': u'bd8a53962fd64ffeab7f2932b0e688c5', u'protected': False, u'id': u'251939be-2d2c-406e-b776-566b7322c928', u'file': u'/v2/images/251939be-2d2c-406e-b776-566b7322c928/file', u'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', u'min_disk': 0, u'virtual_size': None, u'min_ram': 0, u'size': 12716032}}
2018-09-13 19:16:18.348 31485 DEBUG cinder.volume.flows.manager.create_volume [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Cloning 5449e583-f52c-42c3-8617-fd51d9d9a775 from image 251939be-2d2c-406e-b776-566b7322c928  at location (None, None). _create_from_image /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:896
2018-09-13 19:16:18.352 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928/file log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:18.403 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.492s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:18.407 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:18.478 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928/file used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:18.481 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Type: application/octet-stream
Content-Md5: 443b7623e27ecf03dc9e01ee93f67afe
Content-Length: 12716032
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:18 GMT
Connection: keep-alive
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:18.965 31485 DEBUG cinder.image.image_utils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Image fetch details: dest /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm, size 12.00 MB, duration 1.00 sec fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:247
2018-09-13 19:16:18.967 31485 INFO cinder.image.image_utils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Image download 12.00 MB at 12.00 MB/s
2018-09-13 19:16:18.970 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:19.058 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928 used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:19.062 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:19 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1536880201", "tags": [], "container_format": "bare", "created_at": "2018-09-13T23:11:02Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-13T23:11:02Z", "visibility": "public", "self": "/v2/images/251939be-2d2c-406e-b776-566b7322c928", "min_disk": 0, "protected": false, "id": "251939be-2d2c-406e-b776-566b7322c928", "file": "/v2/images/251939be-2d2c-406e-b776-566b7322c928/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "bd8a53962fd64ffeab7f2932b0e688c5", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:19.065 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:19.090 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:19.094 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:19 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:19.104 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:20.137 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.731s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:20.675 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm" returned: 0 in 1.571s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:20.677 31485 DEBUG cinder.image.image_utils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Temporary image 251939be-2d2c-406e-b776-566b7322c928 is fetched for user c1b56a1d5672426f9e3b1fdf25f5593a. fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:672
2018-09-13 19:16:20.679 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:22.108 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm" returned: 0 in 1.429s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:22.112 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvcreate -T -V 1g -n volume-5449e583-f52c-42c3-8617-fd51d9d9a775 cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:23.404 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvcreate -T -V 1g -n volume-5449e583-f52c-42c3-8617-fd51d9d9a775 cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.292s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:23.440 31485 DEBUG cinder.volume.flows.manager.create_volume [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Attempting download of 251939be-2d2c-406e-b776-566b7322c928 ((None, None)) to volume 5449e583-f52c-42c3-8617-fd51d9d9a775. _copy_image_to_volume /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:536
2018-09-13 19:16:23.445 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:23.501 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/images/251939be-2d2c-406e-b776-566b7322c928 used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:23.503 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:23 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1536880201", "tags": [], "container_format": "bare", "created_at": "2018-09-13T23:11:02Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-13T23:11:02Z", "visibility": "public", "self": "/v2/images/251939be-2d2c-406e-b776-566b7322c928", "min_disk": 0, "protected": false, "id": "251939be-2d2c-406e-b776-566b7322c928", "file": "/v2/images/251939be-2d2c-406e-b776-566b7322c928/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "bd8a53962fd64ffeab7f2932b0e688c5", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:23.505 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}098b75e8aec18c06640cd982629146214a816239' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec' http://10.0.20.25:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-13 19:16:23.521 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] GET call to image for http://10.0.20.25:9292/v2/schemas/image used request id req-ae3ace14-df03-4df9-ae83-0f5cc56795ec _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-13 19:16:23.525 31485 DEBUG glanceclient.common.http [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-ae3ace14-df03-4df9-ae83-0f5cc56795ec
Date: Thu, 13 Sep 2018 23:16:23 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-13 19:16:23.534 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpIyrJxO execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:24.797 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpIyrJxO" returned: 0 in 1.263s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:24.801 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:25.873 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm" returned: 0 in 1.073s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:25.876 31485 DEBUG cinder.image.image_utils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] 251939be-2d2c-406e-b776-566b7322c928 was qcow2, converting to raw  fetch_to_volume_format /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:417
2018-09-13 19:16:25.878 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf dd count=0 if=/var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm of=/dev/mapper/cinder--volumes2-volume--5449e583--f52c--42c3--8617--fd51d9d9a775 oflag=direct execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:26.900 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf dd count=0 if=/var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm of=/dev/mapper/cinder--volumes2-volume--5449e583--f52c--42c3--8617--fd51d9d9a775 oflag=direct" returned: 0 in 1.022s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:26.904 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf qemu-img convert -t none -f qcow2 -O raw /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm /dev/mapper/cinder--volumes2-volume--5449e583--f52c--42c3--8617--fd51d9d9a775 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:29.535 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf qemu-img convert -t none -f qcow2 -O raw /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm /dev/mapper/cinder--volumes2-volume--5449e583--f52c--42c3--8617--fd51d9d9a775" returned: 0 in 2.632s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:29.539 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:16:31.160 31485 DEBUG oslo_concurrency.processutils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm" returned: 0 in 1.621s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:16:31.163 31485 DEBUG cinder.image.image_utils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Image conversion details: src /var/lib/cinder/conversion/tmpEVnHl_localhost.localdomain@lvm, size 44.00 MB, duration 2.64 sec, destination /dev/mapper/cinder--volumes2-volume--5449e583--f52c--42c3--8617--fd51d9d9a775 _convert_image /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:190
2018-09-13 19:16:31.164 31485 INFO cinder.image.image_utils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Converted 44.00 MB image at 16.70 MB/s
2018-09-13 19:16:31.166 31485 DEBUG cinder.volume.flows.manager.create_volume [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Downloaded image 251939be-2d2c-406e-b776-566b7322c928 ((None, None)) to volume 5449e583-f52c-42c3-8617-fd51d9d9a775 successfully. _copy_image_to_volume /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:582
2018-09-13 19:16:31.173 31485 DEBUG cinder.image.image_utils [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Temporary image 251939be-2d2c-406e-b776-566b7322c928 for user c1b56a1d5672426f9e3b1fdf25f5593a is deleted. fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:676
2018-09-13 19:16:31.176 31485 DEBUG cinder.volume.flows.manager.create_volume [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Marking volume 5449e583-f52c-42c3-8617-fd51d9d9a775 as bootable. _enable_bootable_flag /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:479
2018-09-13 19:16:31.198 31485 DEBUG cinder.volume.flows.manager.create_volume [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Copying metadata from image 251939be-2d2c-406e-b776-566b7322c928 to 5449e583-f52c-42c3-8617-fd51d9d9a775. _handle_bootable_volume_glance_meta /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:437
2018-09-13 19:16:31.200 31485 DEBUG cinder.volume.flows.manager.create_volume [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Creating volume glance metadata for volume 5449e583-f52c-42c3-8617-fd51d9d9a775 backed by image 251939be-2d2c-406e-b776-566b7322c928 with: {'container_format': u'bare', 'min_ram': 0, 'disk_format': u'qcow2', 'image_name': u'tempest-cirros-0.4.0-x86_64-1536880201', 'image_id': u'251939be-2d2c-406e-b776-566b7322c928', 'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', 'min_disk': 0, 'size': 12716032}. _capture_volume_image_metadata /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:616
2018-09-13 19:16:31.340 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeFromSpecTask;volume:create' (6916758e-a9f1-4b20-a04d-c2a301ef4cc5) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-13 19:16:31.345 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeOnFinishTask;volume:create, create.end' (44b41a81-4387-4cd6-be31-19222f557522) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-13 19:16:31.403 31485 INFO cinder.volume.flows.manager.create_volume [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Volume volume-5449e583-f52c-42c3-8617-fd51d9d9a775 (5449e583-f52c-42c3-8617-fd51d9d9a775): created successfully
2018-09-13 19:16:31.407 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeOnFinishTask;volume:create, create.end' (44b41a81-4387-4cd6-be31-19222f557522) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-13 19:16:31.414 31485 DEBUG cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Flow 'volume_create_manager' (866a8ce4-2c16-4ec7-a5d1-e4c0168a6524) transitioned into state 'SUCCESS' from state 'RUNNING' _flow_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:145
2018-09-13 19:16:31.416 31485 INFO cinder.volume.manager [req-ae3ace14-df03-4df9-ae83-0f5cc56795ec c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Created volume successfully.
2018-09-13 19:17:13.822 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:17:13.825 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:17:13.890 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:17:13.891 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:17:13.894 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:15.519 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.626s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:15.523 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:16.966 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.444s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:16.972 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:18.351 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.379s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:18.355 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:19.611 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.256s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:25.936 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvchange -a y --yes -K cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:27.068 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvchange -a y --yes -K cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775" returned: 0 in 1.132s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:27.160 31485 INFO cinder.volume.targets.lio [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Creating iscsi_target for volume: volume-5449e583-f52c-42c3-8617-fd51d9d9a775
2018-09-13 19:17:27.163 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:17:27.165 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool create /dev/cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775 iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775 6xWFF2eEzakCYALHQ8gu PebEBvNkEDY7DDGC False -p3260 -a10.0.20.25 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:29.627 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool create /dev/cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775 iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775 6xWFF2eEzakCYALHQ8gu PebEBvNkEDY7DDGC False -p3260 -a10.0.20.25" returned: 0 in 2.462s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:29.631 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.467s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:17:29.633 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:17:29.635 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:32.082 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 2.448s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:32.086 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.453s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:17:32.089 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:17:32.091 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:34.564 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.473s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:34.567 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.478s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:17:34.569 31485 DEBUG cinder.volume.targets.iscsi [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Set provider_location to: 10.0.20.25:3260,iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775 iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775 0 create_export /usr/lib/python2.7/site-packages/cinder/volume/targets/iscsi.py:215
2018-09-13 19:17:34.597 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:17:34.598 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool add-initiator iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775 6xWFF2eEzakCYALHQ8gu PebEBvNkEDY7DDGC iqn.1994-05.com.redhat:f393eeb713 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:37.181 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool add-initiator iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775 6xWFF2eEzakCYALHQ8gu PebEBvNkEDY7DDGC iqn.1994-05.com.redhat:f393eeb713" returned: 0 in 2.583s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:37.184 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.587s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:17:37.186 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:17:37.189 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:17:39.539 31485 DEBUG oslo_concurrency.processutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.351s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:17:39.543 31485 DEBUG oslo_concurrency.lockutils [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.357s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:17:39.601 31485 INFO cinder.volume.manager [req-c9a57a1f-2568-410a-ad3e-4f79b625d05f c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Initialize volume connection completed successfully.
2018-09-13 19:17:42.015 31485 DEBUG cinder.coordination [req-35526890-89b8-4e39-b522-6f4c3bac0768 c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "/var/lib/cinder/cinder-5449e583-f52c-42c3-8617-fd51d9d9a775" acquired by "attach_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-09-13 19:17:42.193 31485 INFO cinder.volume.manager [req-35526890-89b8-4e39-b522-6f4c3bac0768 c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Attaching volume 5449e583-f52c-42c3-8617-fd51d9d9a775 to instance 9e18af72-b7c8-49d4-94db-9f59a617a6cd at mountpoint /dev/vdb on host None.
2018-09-13 19:17:42.417 31485 INFO cinder.volume.manager [req-35526890-89b8-4e39-b522-6f4c3bac0768 c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Attach volume completed successfully.
2018-09-13 19:17:42.418 31485 DEBUG cinder.coordination [req-35526890-89b8-4e39-b522-6f4c3bac0768 c1b56a1d5672426f9e3b1fdf25f5593a 33775ae47c434316879560b7bcd0d652 - 2a419c3e93af4d3997e18f6de5d4c119 2a419c3e93af4d3997e18f6de5d4c119] Lock "/var/lib/cinder/cinder-5449e583-f52c-42c3-8617-fd51d9d9a775" released by "attach_volume" :: held 0.403s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-09-13 19:18:13.825 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:18:13.828 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:18:13.886 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:18:13.888 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:18:13.890 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:18:15.145 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.255s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:18:15.149 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:18:16.307 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.158s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:18:16.311 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:18:17.774 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.463s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:18:17.777 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:18:19.153 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.377s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:19:13.828 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:19:13.831 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:19:13.882 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:19:13.883 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:19:13.885 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:19:15.129 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.243s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:19:15.131 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:19:16.484 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.352s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:19:16.487 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:19:17.885 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.398s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:19:17.888 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:19:18.815 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.928s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:13.830 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:20:13.833 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:20:13.873 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:20:13.874 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:20:13.876 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:15.329 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.453s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:15.333 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:16.751 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.419s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:16.756 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:18.141 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.386s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:18.146 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:19.517 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.371s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:27.078 31485 DEBUG oslo_concurrency.lockutils [req-e966537d-22e8-4843-b6f8-6537b2f13a5a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:20:27.084 31485 DEBUG oslo_concurrency.processutils [req-e966537d-22e8-4843-b6f8-6537b2f13a5a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete-initiator iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775 iqn.1994-05.com.redhat:f393eeb713 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:29.997 31485 DEBUG oslo_concurrency.processutils [req-e966537d-22e8-4843-b6f8-6537b2f13a5a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete-initiator iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775 iqn.1994-05.com.redhat:f393eeb713" returned: 0 in 2.914s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:30.000 31485 DEBUG oslo_concurrency.lockutils [req-e966537d-22e8-4843-b6f8-6537b2f13a5a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.922s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:20:30.003 31485 DEBUG oslo_concurrency.lockutils [req-e966537d-22e8-4843-b6f8-6537b2f13a5a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:20:30.005 31485 DEBUG oslo_concurrency.processutils [req-e966537d-22e8-4843-b6f8-6537b2f13a5a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:32.590 31485 DEBUG oslo_concurrency.processutils [req-e966537d-22e8-4843-b6f8-6537b2f13a5a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.586s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:32.593 31485 DEBUG oslo_concurrency.lockutils [req-e966537d-22e8-4843-b6f8-6537b2f13a5a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.590s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:20:32.595 31485 INFO cinder.volume.manager [req-e966537d-22e8-4843-b6f8-6537b2f13a5a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Terminate volume connection completed successfully.
2018-09-13 19:20:33.039 31485 DEBUG cinder.coordination [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "/var/lib/cinder/cinder-5449e583-f52c-42c3-8617-fd51d9d9a775-detach_volume" acquired by "detach_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-09-13 19:20:33.134 31485 INFO cinder.volume.manager [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Detaching volume 5449e583-f52c-42c3-8617-fd51d9d9a775 from instance 9e18af72-b7c8-49d4-94db-9f59a617a6cd.
2018-09-13 19:20:33.138 31485 DEBUG oslo_concurrency.lockutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:20:33.141 31485 DEBUG oslo_concurrency.processutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:35.971 31485 DEBUG oslo_concurrency.processutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 2.831s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:35.975 31485 DEBUG oslo_concurrency.lockutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.837s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:20:35.976 31485 INFO cinder.volume.targets.lio [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Removing iscsi_target: 5449e583-f52c-42c3-8617-fd51d9d9a775
2018-09-13 19:20:35.978 31485 DEBUG oslo_concurrency.lockutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:20:35.981 31485 DEBUG oslo_concurrency.processutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:38.798 31485 DEBUG oslo_concurrency.processutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete iqn.2010-10.org.openstack:volume-5449e583-f52c-42c3-8617-fd51d9d9a775" returned: 0 in 2.818s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:38.802 31485 DEBUG oslo_concurrency.lockutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.823s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:20:38.805 31485 DEBUG oslo_concurrency.lockutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:20:38.808 31485 DEBUG oslo_concurrency.processutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:41.580 31485 DEBUG oslo_concurrency.processutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.773s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:41.583 31485 DEBUG oslo_concurrency.lockutils [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.778s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:20:41.903 31485 INFO cinder.volume.manager [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Detach volume completed successfully.
2018-09-13 19:20:41.905 31485 DEBUG cinder.coordination [req-e221cf00-6260-4d80-b094-62d972cae113 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "/var/lib/cinder/cinder-5449e583-f52c-42c3-8617-fd51d9d9a775-detach_volume" released by "detach_volume" :: held 8.866s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-09-13 19:20:53.014 31485 DEBUG cinder.coordination [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "/var/lib/cinder/cinder-5449e583-f52c-42c3-8617-fd51d9d9a775-delete_volume" acquired by "delete_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-09-13 19:20:53.235 31485 DEBUG oslo_concurrency.lockutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 19:20:53.238 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:56.280 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 3.043s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:56.284 31485 DEBUG oslo_concurrency.lockutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 3.049s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 19:20:56.285 31485 INFO cinder.volume.targets.iscsi [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Skipping remove_export. No iscsi_target is presently exported for volume: 5449e583-f52c-42c3-8617-fd51d9d9a775
2018-09-13 19:20:56.289 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:57.499 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775" returned: 0 in 1.211s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:57.503 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvdisplay --noheading -C -o Attr cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:20:58.806 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvdisplay --noheading -C -o Attr cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775" returned: 0 in 1.304s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:20:58.811 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvremove --config activation { retry_deactivation = 1}  -f cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:21:00.522 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvremove --config activation { retry_deactivation = 1}  -f cinder-volumes2/volume-5449e583-f52c-42c3-8617-fd51d9d9a775" returned: 0 in 1.711s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:21:00.526 31485 INFO cinder.volume.drivers.lvm [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Successfully deleted volume: 5449e583-f52c-42c3-8617-fd51d9d9a775
2018-09-13 19:21:00.631 31485 WARNING cinder.quota [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Deprecated: Default quota for resource: gigabytes_iscsi is set by the default quota flag: quota_gigabytes_iscsi, it is now deprecated. Please use the default quota class for default quota.
2018-09-13 19:21:00.635 31485 WARNING cinder.quota [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Deprecated: Default quota for resource: volumes_iscsi is set by the default quota flag: quota_volumes_iscsi, it is now deprecated. Please use the default quota class for default quota.
2018-09-13 19:21:00.699 31485 DEBUG cinder.quota [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Created reservations ['668c5c18-806d-4f64-a8d2-ac5751f264a4', '3b1a4881-f70e-49d1-843c-9941c0860c65', 'fa4f65b6-fdbe-4b09-bef4-f84f8506f30f', '08658752-5ce4-49ab-b4a9-d5370e29a012'] reserve /usr/lib/python2.7/site-packages/cinder/quota.py:1029
2018-09-13 19:21:00.828 31485 DEBUG cinder.volume.utils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Can not find volume 5449e583-f52c-42c3-8617-fd51d9d9a775 at notify usage _usage_from_volume /usr/lib/python2.7/site-packages/cinder/volume/utils.py:99
2018-09-13 19:21:00.925 31485 DEBUG cinder.volume.drivers.lvm [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:21:00.929 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:21:02.355 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.427s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:21:02.360 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:21:03.593 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.234s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:21:03.596 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:21:04.749 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.153s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:21:04.755 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:21:05.834 31485 DEBUG oslo_concurrency.processutils [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.080s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:21:05.838 31485 DEBUG cinder.manager [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:21:06.043 31485 INFO cinder.volume.manager [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Deleted volume successfully.
2018-09-13 19:21:06.067 31485 DEBUG cinder.coordination [req-78767908-3a16-43a6-a06c-0c84e2e62ad5 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lock "/var/lib/cinder/cinder-5449e583-f52c-42c3-8617-fd51d9d9a775-delete_volume" released by "delete_volume" :: held 13.053s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-09-13 19:21:13.831 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:21:13.833 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:21:13.886 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:21:13.887 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:21:13.889 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:21:14.868 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 0.980s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:21:14.873 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:21:16.199 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.326s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:21:16.203 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:21:17.733 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.530s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:21:17.738 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:21:19.226 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.489s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:22:13.830 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:22:13.833 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:22:13.891 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:22:13.892 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:22:13.895 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:22:15.435 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.541s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:22:15.439 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:22:16.917 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.478s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:22:16.921 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:22:18.175 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.255s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:22:18.180 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:22:19.381 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.202s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:23:13.835 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:23:13.837 31485 DEBUG cinder.manager [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-13 19:23:13.880 31485 DEBUG oslo_service.periodic_task [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-13 19:23:13.881 31485 DEBUG cinder.volume.drivers.lvm [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-13 19:23:13.883 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:23:14.845 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 0.962s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:23:14.849 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:23:16.161 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.312s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:23:16.165 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-13 19:23:17.446 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.282s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-13 19:23:17.452 31485 DEBUG oslo_concurrency.processutils [req-ab801cbc-5e34-4762-b63b-407612a988fc - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
